packages/selinux-policy/selinux-policy/policy/constraints

```diff
@@ -122,6 +122,19 @@ constrain process dyntransition
 	or ( t1 == can_change_process_identity and t2 == process_user_target )
 );
 
+# Constrain access to memory mapping file objects
+# One scenario in which this is useful is for a series of 
+# filters where, once filterA is done processing/modifying it, it passes
+# it to filterB. If filterA had memory mapped the file, it
+# could make changes after filterB had begun processing the file giving filterA
+# the opporunity to send malicious or sensitive information compromising the security
+# goals of the system.
+# Use the "files_not_mappable_file(<type_name>)" interface call to prevent mapping of a file type.
+constrain { dir file blk_file chr_file fifo_file } { map }
+(
+	t2 != not_mappable_file_type
+);
+
```

Comment:
It is just there since constraints have to be in base and this allows modules to add unmappable types. But to answer you question - I can't think of anything off the top of my head that shouldn't be mappable on a standard system.